packages/runner-ct/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@packages/runner-ct",
```

Comment:
Yes, this should not be too hard to clean up once we have a shared-runner package.

This is a decent amount of work that likely should be a separate PR since we will want to go through it carefully. I'd estimate 1.5-2 days of effort.